cmd/lk/room.go

```diff
@@ -177,6 +177,11 @@ var (
 							Usage:     "read attributes from a `JSON` file",
 							TakesFile: true,
 						},
+						&cli.BoolFlag{
+							Name:  "auto-subscribe",
+							Usage: "Automatically subscribe to published tracks. Default is true.",
+							Value: true,
```

Comment:
it should probably be false by default. the CLI client cannot really consume any tracks